src/Writer/CsvWriter.php

```diff
@@ -103,6 +117,10 @@ public function __construct($filename, $delimiter = ',', $enclosure = '"', $esca
     public function open()
     {
         $this->file = fopen($this->filename, 'w', false);
+        if ("\n" !== $this->terminate) {
+            stream_filter_register('filterTerminate', 'Exporter\Writer\CsvWriterTerminate');
```

Comment:
can you replace `'Exporter\Writer\CsvWriterTerminate'` with `CsvWriterTerminate::class`